R/utils.R

```diff
@@ -125,35 +75,33 @@ gama_remote_distrib <- function() {
                             options("rama.default.gama.osx")),
          "Windows" = paste0(options("rama.repo"),
                             options("rama.default.gama.win64")),
-         # to complete C:\Program Files\
          "Linux"   =  paste0(options("rama.repo"),
                              options("rama.default.gama.linux")))
-  # to complete
 }
 
 # Downloads gama ---------------------------------------------------------------
 #' @importFrom utils download.file unzip untar
 #' @importFrom downloader download
 download_gama <- function() {
   distrib <- gama_remote_distrib()
-  expDir  <- gama_local_distrib_path();
+  expDir  <- gama_local_distrib_path()
   path <- paste0(options("rama.temp_dir"), "/")
   path_dist <- paste0(path, "out/", basename(expDir))
   path_test <- dirname(expDir)
   distrib_file <- paste0(path, "downloaded_gama.tgz")
-  print("coucou");
-  print(path);
-
-
-  if (! dir.exists(path)) dir.create(path, recursive = TRUE)
+  print(path)
```

Comment:
What is the use of this command?

Reply:
I don't think the users need it. It's not my code but I think it's a print to check if the path is correct so I agree to remove it.

Reply:
ok